src/towncrier/build.py

```diff
@@ -263,12 +263,12 @@ def __main(
         click.echo("Staging newsfile...", err=to_err)
         _git.stage_newsfile(base_directory, news_file)
 
-        click.echo("Removing news fragments...", err=to_err)
         if should_remove_fragment_files(
```

Comment:
A good additional fix would be changing this line
```suggestion
        if fragment_filenames and should_remove_fragment_files(
```
Currently you get the following process if there are no files to remove:
```
Removing news fragments...
Removing the following files:
Is it okay if I remove those files? [Y/n]
```

Reply:
Nice! I moved the check inside `should_remove_fragment_files`, instead of doing it outside.